user-guide/Advanced_Functionality/Databases/Indexing_database/OpenSearch_Database/Installing_OpenSearch_database.md

```diff
@@ -267,21 +236,72 @@ plugins.security.system_indices.enabled: true
 plugins.security.system_indices.indices: [".plugins-ml-model", ".plugins-ml-task", ".opendistro-alerting-config", ".opendistro-alerting-alert*", ".opendistro-anomaly-results*", ".opendistro-anomaly-detector*", ".opendistro-anomaly-checkpoints", ".opendistro-anomaly-detection-state", ".opendistro-reports-*", ".opensearch-notifications-*", ".opensearch-notebooks", ".opensearch-observability", ".opendistro-asynchronous-search-response*", ".replication-metadata-store",".opendistro-anomaly-detection-state", ".opendistro-reports-*", ".opensearch-notifications-*", ".opensearch-notebooks", ".opensearch-observability", ".opendistro-asynchronous-search-response*", ".replication-metadata-store"]
 ```
 
+##### Change the admin password and remove other users
+
+Generate a new hash for the admin user as detailed under [Configure a user](https://opensearch.org/docs/latest/install-and-configure/install-opensearch/debian/#configure-a-user) in the OpenSearch documentation. You will need to remove all the users except *admin* and replace its hash with the one generated from the new password:
+
+1. Generate a hash for the new password of the *admin* user:
```

Comment:
This isn't entirely clear to me. Do you need to create a new password first before you do this (if yes, this should still be added to the procedure), or is the creation of the new password included in this step?

Reply:
Indeed, the new password is created in step 1. where a hash key is generated from the new password chosen at the time, and then in step 2., this hash key is added to the yaml file, replacing the old one there.
Basically here we are changing the default password, not entering it in clear text but instead, adding its corresponding hash key. Steps 1. and 2. try to describe this.